### DISCUSSION TITLE
posts/ptatm-1/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#14] [Announcements]
# posts/ptatm-1/

本文介绍适用于多路径任务的多核系统pWCET分析工具——PTATM的初始化工作，如内核、环境变量等。

https://www.haoyep.com/posts/ptatm-1/

**Comment by panxingyuan1** [comment]
请问这个PTATM是干嘛的呀，能测量操作系统中任务执行时间的分布吗

**Reply by leegical** [reply]
只能测量单个二进制程序的执行时间分布，且这个程序能被angr分析。目前限制比较多，直接用在实际应用中有点困难
